lib/types/array.js

```diff
@@ -610,29 +610,12 @@ module.exports = Any._extend({
 
 internals.fillMissedErrors = function (schema, errors, requireds, value, state, prefs) {
 
-    const knownMisses = [];
     let unknownMisses = 0;
-    for (const required of requireds) {
-        const label = required._flags.label;
-        if (label) {
-            knownMisses.push(label);
-        }
-        else {
-            ++unknownMisses;
-        }
+    for (let i = 0; i < requireds.length; ++i) {
```

Comment:
@zemccartney couldn't this have stayed a `for of` loop?

Reply:
yea, that would have worked no problem, functionally, but IIRC, that triggered an unused var linting error (we don't reference any values from `requireds` in the loop body). I felt switching to a `for` loop was cleaner than a `for of` plus a linter disabling comment, but that came down to a more or less random and maybe inconsistent stylistic decision. happy to change if the group feels a `for or` is preferable here!